scripts/nb-tester/test-notebook.py

```diff
@@ -214,15 +243,12 @@ def create_argument_parser() -> argparse.ArgumentParser:
     args = create_argument_parser().parse_args()
 
     paths = map(Path, args.filenames or find_notebooks(submit_jobs=args.submit_jobs))
-    if not args.submit_jobs:
-        paths = [path for path in paths if not any(path.match(glob) for glob in NOTEBOOKS_THAT_SUBMIT_JOBS)]
+    paths = filter_paths(paths, submit_jobs=args.submit_jobs)
```

Comment:
Nit: you're changing the variable type of `paths` here, which is subtle and discouraged in Python. (Fine in other languages like Rust.) Consider calling it `specified_paths` and `filtered_paths`.

Reply:
Sounds good, thanks!